lib/ProductOpener/Lang.pm

```diff
@@ -35,8 +35,7 @@ through the lang() and lang_sprintf() functions.
 
 package ProductOpener::Lang;
 
-use utf8;
-use Modern::Perl '2017';
+use ProductOpener::PerlStandards
```

Comment:
You missed a column here @dipietroR 

I imediatly commit it to see if it fixes the checks.

```suggestion
use ProductOpener::PerlStandards;
```